tests/suites/controller/enable_ha.sh

```diff
@@ -0,0 +1,62 @@
+wait_for_controller_machines() {
+    ammount=${1}
+
+    attempt=0
+    until [[ "$(juju machines -m controller --format=json | jq -r ".machines | .[] | .[\"juju-status\"] | select(.current == \"started\") | .current" | wc -l | grep "${ammount}")" ]]; do
```

Comment:
This ensures that the machines have started.  Check for ha-status = ha-enabled on all 3 machines to ensure HA is up and running.

$ juju show-controller
  controller-machines:
    "0":
      instance-id: juju-a2cba4-0
      ha-status: ha-enabled
      ha-primary: true
    "1":
      instance-id: juju-a2cba4-1
      ha-status: ha-enabled
    "2":
      instance-id: juju-a2cba4-2
      ha-status: ha-enabled

Reply:
Or "controller-member-status": "has-vote" from juju machines.

Reply:
Do we care for this test? I know what you mean in the general case, but for this test we know we'll have exactly 3 machines for a controller...?

Reply:
We do, just because the machines deploy, doesn't mean that enable HA was successful.  The delays in the field have been after the machines were started.